docs/source/guide/project_settings_lse.md

```diff
@@ -376,33 +387,30 @@ To assign a project-level role, first add the person to your project. Once added
-| **Show predictions to annotators in the Label Stream and Quick View** | When enabled, predictions are shown to annotators during the labeling process. This is enabled by default.<br /><br />Disable this option to hide predictions from annotators. For example, you might want to hide predictions to prevent bias. |
+| **Start model training on annotation submission**         | Triggers the connected ML backend to start the training process each time an annotation is created or updated. <br /><br />This is part of an [active learning loop](active_learning) where the model can be continuously improved as new annotations are added to the dataset. When this setting is enabled, the ML backend's `fit()` method is called, allowing the model to learn from the most recent annotations and potentially improve its predictions for subsequent tasks.   |
+| **Interactive preannotations**         | (Available when creating or editing a model connection)<br /><br />Enable this option to allow the model to assist with the labeling process by providing real-time predictions or suggestions as annotators work on tasks.  <br /><br />In other words, as you interact with data (for example, by drawing a region on an image, highlighting text, or asking an LLM a question), the ML backend receives this input and returns predictions based on it.   |
 
-</dd>
 
-<dt>Model Version</dt>
+And the following actions are available from the overflow menu next to a connected model:
 
-<dd>
-
-If you have multiple versions, you can select which version is used to generate predictions. 
+| Action          | Description    |
+| ------------- | ------------ |
+| **Start Training**         | Manually initiate training. Use this action if you want to control when the model training occurs, such as after a specific number of annotations have been collected or at certain intervals.  |
+| **Send Test Request**         | (Available from the overflow menu next to the connected model)<br /><br />Use this for troubleshooting and sending a test resquest to the connected model.   |
+| **Edit**         | Edit the model name, URL, and parameters. For more information, see [Connect a model to Label Studio](ml#Connect-a-model-to-Label-Studio). |
+| **Delete**         | Remove the connection to the model. |
 
-</dd>
+## Predictions
 
-</dl>
+From here you can view predictions that have been imported or generated through training. For more information, see [Import pre-annotated data into Label Studio](predictions). 
```

Comment:
`...generated through training...` - that's not correct. I assume the idea is about "generated in batch via 'Retrieve Predictions' or 'Batch Predictions'" action called when selecting the task in Data Manager. We need to fix this elsewhere, and ensure we have the workflow described, and the link provided

Reply:
See changes - not sure if I captured this correctly